scheduler/app/libs/tasks/get_assets.py

```diff
@@ -126,12 +135,28 @@ def augment_total_balance(
 
 
 async def build_treasury_with_assets(
-    treasury_address: str, chain_id: int, start: str, end: str
+    treasury_address: str, chain_id: int, start: str = None, end: str = None
 ) -> tuple[Treasury, dict[str, DF], dict[str, DF]]:
     treasury = filter_out_small_assets(await make_treasury(treasury_address, chain_id))
     augmented_token_hist_prices = augment_token_hist_prices(
         await get_token_hist_prices(treasury)
     )
+
+    if not start:
+        from datetime import datetime
```

Comment:
In terms of the imports, let's keep them always at start of the file. When we'll introduce a linter, this code will not pass. Also, you don't need mktime here.